src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -61,8 +61,9 @@ public boolean canEnchantItem(ItemStack item) {
             final ItemMeta itemMeta = item.getItemMeta();
             final Optional<String> id = SlimefunPlugin.getItemDataService().getItemData(itemMeta);
             if (id.isPresent()) {
-                return id.get().equals(Items.ADVANCED_CIRCUIT.getItemId())
-                    || id.get().equals(Items.NANO_BLADE.getItemId());
+                return (id.get().equals(Items.ADVANCED_CIRCUIT.getItemId()))
+                    || (id.get().equals(Items.NANO_BLADE.getItemId()))
+                    || (id.get().equals(Items.GLASS_CUTTER.getItemId()));
```

Comment:
make this a var since we will use it a lot

Reply:
Cant, the enchants are initialized before the items